apps/playwright/tests/admin/admin-workflows.spec.ts

```diff
@@ -0,0 +1,114 @@
+import AppPage from "~/fixtures/AppPage";
+import { test, expect } from "~/fixtures";
+import { loginBySub } from "~/utils/auth";
+
+const USERS_PAGINATED_QUERY = "UsersPaginated";
+
+const searchForUser = async (appPage: AppPage, name: string) => {
+  await appPage.page.getByRole("textbox", { name: /search/i }).fill(name);
+
+  await appPage.waitForGraphqlResponse(USERS_PAGINATED_QUERY);
+};
+
+const goToUsersPage = async (appPage: AppPage) => {
+  await appPage.page.goto("/en/admin/users");
+  await appPage.waitForGraphqlResponse(USERS_PAGINATED_QUERY);
+};
+
+test.describe("Admin workflows", () => {
+  test.beforeEach(async ({ appPage }) => {
+    await loginBySub(appPage.page, "admin@test.com");
+  });
+
+  test("Searching for user and reviewing the profile", async ({ appPage }) => {
```

Comment:
```suggestion
  test("Search for user and review profile", async ({ appPage }) => {
```

Reply:
So I actually changed these to read like a sentence like the `it()` did. So it reads like:

> Test searching for user and reviewing the profile

Or, do we want them to just plainly state what occurs in the test? :thinking:

Reply:
The [Playwright docs](https://playwright.dev/docs/writing-tests#first-test) use present simple tense and I think a lot of the other tests that already exist in our codebase in Playwright also use present simple tense instead of the present participle. Shorter the better?